src/source/PeerConnection/Rtcp.c

```diff
@@ -102,6 +110,20 @@ STATUS onRtcpPacket(PKvsPeerConnection pKvsPeerConnection, PBYTE pBuff, UINT32 b
         }
 
         currentOffset += (rtcpPacket.payloadLength + RTCP_PACKET_HEADER_LEN);
+
+        if (pTransceiver == NULL)
```

Comment:
Couple of comments.

* Please use curlies even with a single-line statements
* Strongly recommend not using control flow modification keywords within these large loops. In this case, you could simply do the if (pTransceiver != NULL) { do the reporting }